pom.xml

```diff
@@ -388,6 +388,21 @@
                 <artifactId>bcutil-jdk15on</artifactId>
                 <version>1.70</version>
             </dependency>
+            <dependency>
+                <groupId>org.apache.ant</groupId>
```

Comment:
What's the reasoning behind adding new deps ?

Reply:
This isn't adding new deps, just pinning the versions to avoid vulnerable components.

Reply:
can we add some comments to separate those dependency declarations from the ones we depend on directly? We should also note which dependencies pull in those affected versions. We'll want to remove them as soon as possible to avoid pinning to older versions whenever the direct dependencies upgrade or remove the affected ones.

Reply:
It would be worthwhile to separate sections of direct and transitive dependencies, as well as identify the dependencies that belong to specific modules. 
Adding clarifications to the dependencies specification that was added.